src/m365/spo/commands/list/list-remove.spec.ts

```diff
@@ -147,6 +147,55 @@ describe(commands.LIST_REMOVE, () => {
     assert(correctRequestIssued);
   });
 
+  it('recycles the list when prompt confirmed', async () => {
+    sinon.stub(request, 'post').callsFake((opts) => {
+      requests.push(opts);
+
+      if ((opts.url as string).indexOf(`/recycle()`) > -1) {
+        if (opts.headers &&
+          opts.headers.accept &&
+          (opts.headers.accept as string).indexOf('application/json') === 0) {
+          return Promise.resolve();
+        }
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    sinonUtil.restore(cli.promptForConfirmation);
+    sinon.stub(cli, 'promptForConfirmation').resolves(true);
+    await command.action(logger, { options: { id: 'b2307a39-e878-458b-bc90-03bc578531d6', webUrl: 'https://contoso.sharepoint.com', recycle: true } });
+    let correctRequestIssued = false;
+    requests.forEach(r => {
+      if (r.url.indexOf(`/recycle()`) > -1 &&
+        r.headers.accept &&
+        r.headers.accept.indexOf('application/json') === 0) {
+        correctRequestIssued = true;
+      }
+    });
```

Comment:
Checking for these headers doesn't add much value in my opinion.

Reply:
Sure. Removed those headers